azkaban-common/src/main/java/azkaban/executor/ServerStatistics.java

```diff
@@ -0,0 +1,163 @@
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.executor;
+
+import java.util.Date;
+import java.util.Map;
+
+
+ /** Class that exposes the statistics from the executor server.
+  *  List of the statistics -
+  *  remainingMemoryPercent;
+  *  remainingMemory;
+  *  remainingFlowCapacity;
+  *  numberOfAssignedFlows;
+  *  lastDispatchedTime;
+  *  cpuUsage;
+  *
+  * */
+  public class ServerStatistics {
+    private double remainingMemoryPercent;
+    private long   remainingMemory;
+    private int    remainingFlowCapacity;
+    private int    numberOfAssignedFlows;
+    private Date   lastDispatchedTime;
+    private double cpuUsage;
+
+    public double getCpuUsage() {
+      return this.cpuUsage;
+    }
+
+    public void setCpuUpsage(double value){
+      this.cpuUsage = value;
+    }
+
+    public double getRemainingMemoryPercent() {
+      return this.remainingMemoryPercent;
+    }
+
+    public void setRemainingMemoryPercent(double value){
+      this.remainingMemoryPercent = value;
+    }
+
+    public long getRemainingMemory(){
+      return this.remainingMemory;
+    }
+
+    public void setRemainingMemory(long value){
+      this.remainingMemory = value;
+    }
+
+    public int getRemainingFlowCapacity(){
+      return this.remainingFlowCapacity;
+    }
+
+    public void setRemainingFlowCapacity(int value){
+      this.remainingFlowCapacity = value;
+    }
+
+    public Date getLastDispatchedTime(){
+      return this.lastDispatchedTime;
+    }
+
+    public void setLastDispatchedTime(Date value){
+      this.lastDispatchedTime = value;
+    }
+
+    public int getNumberOfAssignedFlows () {
+      return this.numberOfAssignedFlows;
+    }
+
+    public void setNumberOfAssignedFlows (int value) {
+      this.numberOfAssignedFlows = value;
+    }
+
+    public ServerStatistics(){}
+
+    public ServerStatistics (double remainingMemoryPercent,
+        long remainingMemory,
+        int remainingFlowCapacity,
+        Date lastDispatched,
+        double cpuUsage,
+        int numberOfAssignedFlows){
+      this.remainingMemory = remainingMemory;
+      this.cpuUsage = cpuUsage;
+      this.remainingFlowCapacity = remainingFlowCapacity;
+      this.remainingMemoryPercent = remainingMemoryPercent;
+      this.lastDispatchedTime = lastDispatched;
+      this.numberOfAssignedFlows = numberOfAssignedFlows;
+    }
+
+    @Override
+    public boolean equals(Object obj)
+    {
+        if (obj instanceof ServerStatistics)
```

Comment:
if obj is null instanceof will return false, which is expected and we will return false here for the equals check. 

Please refer to http://docs.oracle.com/javase/specs/jls/se7/html/jls-15.html#jls-15.20.2 for detauls,